src/ts/waterfall/details-overlay/html-details-body.ts

```diff
@@ -129,7 +138,7 @@ export function createDetailsBody(requestID: number, entry: WaterfallEntry, acco
       </div>
       ${timingsTab}
       <div class="tab raw-data">
-        <pre><code>${JSON.stringify(entry.rawResource, null, 2)}</code></pre>
+        <pre><code>${escapeHtml(JSON.stringify(entry.rawResource, null, 2))}</code></pre>
```

Comment:
to make it clear, I am talking about `${responseDl}` and `${responseHeadersDl}` and others above.

you are right. only a forged payload could make the above - not yet escaped rendering - beeing a XSS target.